helpers/utils.js

```diff
@@ -1,19 +1,238 @@
+const { Logger } = require('@hmcts/nodejs-logging');
+const requestModule = require('request-promise-native');
+const request = requestModule.defaults({ 'proxy': 'http://proxyout.reform.hmcts.net:8080' });
+const fs = require('fs');
+
+const logger = Logger.getLogger('helpers/utils.js');
+
 const getSolicitorLoginDetails = () => {
     if (!process.env.CCD_E2E_PASSWORD) {
         throw new Error('You need to set CCD_E2E_EMAIL and CCD_E2E_PASSWORD env variables');
     }
     return {
-        username: 'michael.osinloye@hmcts.net' ,
+        username: process.env.CCD_E2E_EMAIL,
         password: process.env.CCD_E2E_PASSWORD
     };
 }
 
+const getCaseWorkerLoginDetails = () => {
+    if (!process.env.CCD_E2E_PASSWORD) {
```

Comment:
What if username is missing?

Reply:
Since this should be passed in by the piepline as a nightly test, the tests should just fail.
Having error message may help debugging I suppose.